### ISSUE TITLE
Official reviews

### ISSUE THREAD

**Issue by fmaussion** [OGGM/EarthCube2021#6]
Hi Team, here are the reviews I received per mail about our notebook:

**Reviewer comments**: This notebook is very ambitious in what it covers and it runs very nicely. The tools demonstrated are clearly useful and informative. However, there is too much content and it is hard to follow without more introductions or explanation in the text: this many figures are a real challenge for anyone to digest. I understand that the user is supposed to infer the answers to the questions from the output of the notebook, but more explanation of each result would be helpful. The section about correcting for "hydrological months" was not understandable to me: I suggest either cutting it or adding more explanation. Also the graphic from Huss &Hock isn't available.

**Reviewer comments**: this notebook provided an opportunity to learn glacier mass balance concepts and modeling to help understand glacier contribution to annual streamflow. The notebook is quite long and covers a lot of material that should probably ideally be separated into many smaller modules. Overall an incredible resource for introducing glacier modeling. A solid proofread for formatting consistency( () vs []), correct links (e.g. pandas), typos, and acronym definitions (Mt, CMIP) would be helpful. The figures currently have no alt text, which is an accessibility issue. I'm also wondering if it would be possible to more clearly highlight some of the locations that students should change parameters. Since some of the plots are created multiple times in different scenarios, it would reduce notebook length to provide a few helper plotting functions; I really liked the consistency in the plots throughout the notebook, because it made it easier to compare/contrast the various model outputs.

**Reviewer comments**: The notebook presents the Open Global Glacier Model (OGGM-Edu) as an education platform to demonstrate glacier behavior. It first demonstrates basic concepts of glacier dynamics with conceptual examples. Then it applies the model to real world data. The notebook is organized in a logical manner that help user build understanding of glaciers and the model from the basics to being able to apply the model to address scientific questions. With some minor changes and additional explanations, it is suitable for general audience. Missing data import and data processing and analysis portions of the template. While the format of the current notebook makes sense and flows well logically, subsections of “data import and data processing and analysis” can be added to the later examples that contain more detailed explanations of the data used and how the downloaded glacier and climate data are processed to produce model input.

Non-working link to 'mass-balance notebook' in the Glacier mass balance section. Please check other links as well.

Please add more dedicated explanations of figures and key graphs, such as with markdown alt texts.

The statement '1. When a glacier is in equilibrium, a glacier does not contribute to the annual runoff at all.' at the start of part 2 is confusing at first. If not runoff, where does the melted water from the glacier go? When making the point that total mass of the glacier is not changing, perhaps using ‘water balance’ or ‘storage’ is better than ‘runoff’. Make it immediately clear that input-output=0, not output=0.

**Maussion-total**: accept with minor revisions

I think that the reviews make sense, in particular that i may be too ambitious. I would rather not change too much at this stage, but maybe remove what is too hard to understand or add explanations, links to other resources, etc. 

After modifications, the notebook will be accepted into some sort of proceedings, I'm not sure exactly how this will look like.

We need to address the reviews above  by **June 11th**

**Comment by fmaussion** [comment]
TODO list:

- [ ] Make Huss & Hock graphic available
- [ ] Proofread and correct links
- [ ] etc.

**Comment by Holmgren825** [comment]
What I've done in https://github.com/OGGM/EarthCube2021/commit/5ca998093d8d65f723e065d9b32d979221343a00
- Typos
- Fixed broken links
- Added links to numpy, pandas etc.
- Tried to address the problem one reviewer had with the hydrological months with a short explanation. Maybe take a look if you agree with it.
- Similar as above but for the problem with "When a glacier is in equilibrium, a glacier does not contribute to the annual runoff at all." Same here, maybe take a quick look.
- Acronym definitions (CMIP, Mt).

We have alt texts for images but not sure if it is possible to add this to plots?

**Comment by fmaussion** [comment]
> maybe take a quick look

I'll do this in the very end, maybe some others will have an opinion on this as well.

> We have alt texts for images but not sure if it is possible to add this to plots?

No but that's not necessary, the alt-text are for images we fetch from the web.

**Comment by pat-schmitt** [comment]
What I've done in https://github.com/OGGM/EarthCube2021/pull/8/commits/9c12e038fbe981080aef26c7a86fcd53694cbc78:
- changed some x labels
- included a paragraph mentioning the different preprocessed glacier directories and the raw data behind
- added a link for CMIP5

**Comment by ehultee** [comment]
@Holmgren825 note I also tweaked your phrasing in the hydro section to reflect "net contribution" to runoff - I think this solves the reviewer comment.

**Comment by fmaussion** [comment]
Thanks so much! I'm a bit annoyed that all these improvements are here now and will be tedious to feed back to the original notebooks on OGGM-Edu. I'm starting to wonder if this EarthCube thing was a good idea after all...

**Comment by fmaussion** [comment]
> The question about "mass balance feedback" is also too specialized...I guess this is the elevation/MB feedback that stabilizes retreating mountain glaciers?

No, this is the positive feedback between grid point elevation and mass-balance (in the accumulation area: thicker ice columns are higher and therefore colder and therefore accumulate more - in the ablation area: thinner ice columns are lower and melt more) . It is mostly used in the ice sheet community (but also valid for glaciers). I did find some resources but are scattered, e.g. https://atmos.uw.edu/~bitz/514_2013/lecture_may2.pdf

I think that we need to write to Bethan about adding AAR and mass-balance elevation feedback to the glossary: 

www.antarcticglaciers.org/glaciers-and-climate/numerical-ice-sheet-models/numerical-modelling-glossary

**Comment by fmaussion** [comment]
Ive added some definitions to the notebook:

*Tip: some definitions*

- Accumulation-area ratio (AAR): The ratio, often expressed as a percentage, of the area of the accumulation zone to the area of the glacier. The AAR is bounded between 0 and 1. On many glaciers it correlates well with the climatic mass balance. The likelihood that the climatic mass balance will be positive increases as the AAR approaches 1. (from the [Glossary of glacier mass balance and related terms](https://wgms.ch/downloads/Cogley_etal_2011.pdf))
- Mass-balance elevation feedback: a positive feedback which links ice elevation with mass balance. When ice thickens, its surface rises to higher elevations, which are colder and may receive more solid precipitation, hence thickening further. The same feedback also applies to the thinning case ("positive feedback" occurs when an inital perturbation is enhanced further by the feedback processes).

**Comment by ehultee** [comment]
>No, this is the positive feedback between grid point elevation and mass-balance (in the accumulation area: thicker ice columns are higher and therefore colder and therefore accumulate more - in the ablation area: thinner ice columns are lower and melt more) .

Hah, okay, so that's another version of the same effect I was thinking.  The stabilizing effect on retreating mountain glaciers also comes from higher elevation=higher MB, in that the mean elevation of a glacier whose tongue has retreated will go up.

One way to frame it without a lot of added definitions--maybe for the edu notebooks now that you've done it in the EarthCube NB--is to ask something like: "A feedback occurs when one change in the system brings about other changes.  Given what you know about mass balance gradient, what feedbacks can you imagine between glacier change (ice thinning, thickening, glacier growing, glacier retreating) and mass balance?"

Then both answers are acceptable 🙃 

I know that I am making a suggestion in discussion instead of a PR and that's annoying.  If you like the open-ended (or don't care) I can plan to tweak the edu notebooks a bit later; if not then we can leave it alone.
